components/kyma-environment-broker/internal/process/provisioning/enable_for_trial_step.go

```diff
@@ -31,12 +31,7 @@ func (s *EnableForTrialPlanStep) Name() string {
 }
 
 func (s *EnableForTrialPlanStep) Run(operation internal.ProvisioningOperation, log logrus.FieldLogger) (internal.ProvisioningOperation, time.Duration, error) {
```

Comment:
few steps, like this one, don't need operationManager anymore. Create an issue (or next PR) to clean it up or do it in this PR

Reply:
thanks, I did it now